python/ambassador/ir/ir.py

```diff
@@ -976,12 +863,6 @@ def features(self) -> Dict[str, Any]:
         od['endpoint_routing_envoy_maglev_count'] = endpoint_routing_envoy_maglev_count
         od['endpoint_routing_envoy_lr_count'] = endpoint_routing_envoy_lr_count
 
-        cluster_ingresses = self.aconf.get_config("ClusterIngress")
-        od['cluster_ingress_count'] = len(cluster_ingresses.keys()) if cluster_ingresses else 0
-
-        knative_ingresses = self.aconf.get_config("KnativeIngress")
-        od['knative_ingress_count'] = len(knative_ingresses.keys()) if knative_ingresses else 0
-
```

Comment:
It would definitely be a good thing to preserve this count somehow. Maybe just keep a countin the `aconf`?

Reply:
Sounds good. I'll figure out how to bring it back. :smile:

Reply:
Thanks!